0-bootstrap/main.tf

```diff
@@ -108,3 +108,18 @@ module "seed_bootstrap" {
   sa_org_iam_permissions = []
 }
 
+module "gcp_projects_state_bucket" {
```

Comment:
if you moved the `gcp_projects_state_bucket` to the `main.tf` file you will also need to remove it from the GitHub file
https://github.com/renato-rudnicki/terraform-google-enterprise-genai/blob/fb17848e30caad5ecf1e2755de156415d734e893/0-bootstrap/github.tf.example#L111